src/index.js

```diff
@@ -140,6 +148,15 @@ export const init = (opts) => {
     setOptions (changedOptions) {
       const oldOptions = this.options
       this.options = formatOptions({...this.options,...changedOptions});
+
+      const jwt_urls = fetchUrlsFromJWT(changedOptions.token)
+      if (typeof jwt_urls === "undefined") {
+        console.log("WARN: Using JWT that does not specify urls")
+      }
+      else {
+        options.urls = {...jwt_urls, ...changedOptions.urls}
```

Comment:
By the way, this won't override the old options. When you want to use the object spread to override values you have to do it like this `{...oldValues, ...newValues}`